connections/fluid.lua

```diff
@@ -123,7 +123,7 @@ local function transfer(from, to, from_cap, to_cap)
 	local to_box = to_boxes[1]
 	if from_box ~= nil then
 		if to_box == nil then 
-			if from_box.amount < to_cap then
+			if from_box.amount <= to_cap then
```

Comment:
I'm not sure whether this is an intended change, it's at least not related to porting the code to Factorio v 0.17.

Reply:
This was necessary due to the fluid handling changes in 0.17, the game will otherwise crash when placing a "factory input pipe" inside a factory. I believe it should have been like this prior to 0.17 though so I made it a separate commit to make note of it.